cli/cmd/uninstall.go

```diff
@@ -143,6 +151,25 @@ func UninstallClusterResources(ctx context.Context, client *kube.Client, ns stri
 
 }
 
+func waitForPodsToRolloutWithoutInstrumentation(ctx context.Context, client *kube.Client) {
+	l := log.Print("Rolling back odigos changes to pods...")
+	instrumentedPodReq, _ := k8slabels.NewRequirement(k8sconsts.OdigosAgentsMetaHashLabel, selection.Exists, []string{})
+	wait.PollImmediate(1*time.Second, 5*time.Minute, func() (bool, error) {
```

Comment:
looks like [it's deprecated](https://pkg.go.dev/k8s.io/apimachinery/pkg/util/wait#PollImmediate) in favor of `PollUntilContextTimeout`:

```
Deprecated: This method does not return errors from context, use PollUntilContextTimeout. Note that the new method will no longer return ErrWaitTimeout and instead return errors defined by the context package. Will be removed in a future release.
```